deployment/aks-periscope.yaml

```diff
@@ -115,8 +118,8 @@ metadata:
   namespace: aks-periscope
 type: Opaque
 data:
-  AZURE_BLOB_ACCOUNT_NAME: # <accountName, base64 encoded>
-  AZURE_BLOB_SAS_KEY: # <saskey, base64 encoded>
+  AZURE_BLOB_ACCOUNT_NAME: YXpzdG9yYWdlYWNjam9oc2g=
```

Comment:
⚠️ Due to security reasons I would recommend cycling this account name as well.

Reply:
Thanks for pointing this out! Removed and deleted.